src-tauri/src/commands.rs

```diff
@@ -714,32 +671,32 @@ pub async fn get_transaction_history(
 pub async fn import_seed_words(
     seed_words: Vec<String>,
     _window: tauri::Window,
-    state: tauri::State<'_, UniverseAppState>,
     app: tauri::AppHandle,
 ) -> Result<(), String> {
     let timer = Instant::now();
     let config_path = app
-        .path_resolver()
+        .path()
         .app_config_dir()
         .expect("Could not get config dir");
     let data_dir = app
-        .path_resolver()
+        .path()
         .app_local_data_dir()
         .expect("Could not get data dir");
 
-    stop_all_processes(state.inner().clone(), false).await?;
+    stop_all_processes(app.clone(), false).await?;
 
     tauri::async_runtime::spawn(async move {
         match InternalWallet::create_from_seed(config_path, seed_words).await {
             Ok(_wallet) => {
-                InternalWallet::clear_wallet_local_data(data_dir).await?;
+                InternalWallet::clear_wallet_local_data(data_dir)
+                    .await
+                    .map_err(|e| e.to_string())?;
                 info!(target: LOG_TARGET, "[import_seed_words] Restarting the app");
                 app.restart();
-                Ok(())
             }
             Err(e) => {
                 error!(target: LOG_TARGET, "Error loading internal wallet: {:?}", e);
-                Err(e)
+                Err::<(), std::string::String>(e.to_string())
```

Comment:
👀 
did this not work?

```suggestion
                Err(e.to_string())
```